aws/service.ts

```diff
@@ -274,8 +274,8 @@ function computeImageFromBuild(
     pulumi.log.debug(`Building container image at '${build}'`, repository);
     const { repositoryUrl, registryId } = repository;
 
-    return pulumi.all([repositoryUrl, registryId]).apply(([repoUrl, regId]) =>
-        computeImageFromBuildWorker(preEnv, build, imageName, repoUrl, regId, repository));
+    return pulumi.all([repositoryUrl, registryId]).apply(([repositoryUrl, registryId]) =>
+        computeImageFromBuildWorker(preEnv, build, imageName, repositoryUrl, registryId, repository));
```

Comment:
i'd really like the pattern to be:  you just pass in the name of the Outputs to pulumi.all, then you continue using those names on the 'inside', except now they are the unwrapped values.

Having to continuously come up with similar, but slightly different names here is def annoying.